packages/persist/src/index.ts

```diff
@@ -125,7 +125,7 @@ export const reatomPersist = (
         ctx: Ctx,
         state: AtomState<T>,
       ): PersistRecord => ({
-        data: toSnapshot(ctx, state),
+        data: toSnapshot({ ...ctx, spy: undefined }, state),
```

Comment:
You could remove it, the bug is fixed here:https://github.com/artalar/reatom/commit/344d2ce10676f49999c8c3fb973109e1ac42c57c